Tests/MarigoldSwiftTests/Marigold+ConcurrencySpec.swift

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2022 Sailthru
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files
+// (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge,
+// publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+// THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+// CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+// DEALINGS IN THE SOFTWARE.
+
+import XCTest
+@testable import Marigold
+@testable import MarigoldSwift
+
+enum TestErrors : Error {
+    case testError
+}
+
+final class MarigoldConcurrencyTests: XCTestCase {
+    var subject = DummyMarigold()
+    
+    override func setUp() {
+        subject = DummyMarigold()
+    }
+
+    // MARK: Device
+    
+    func test_clearDeviceDataForTypes_callsCorrectMethod() async throws {
```

Comment:
Looks like the test for error handling is missing. Also, is this test passing locally for you? I would expect it to fail because the `clear` method is missing from the `DummyMarigold` class.